src/pages/tutorials/create-an-nft/index.md

```diff
@@ -1,533 +1,521 @@
+   To stop the container, run the command `tznft kill-sandbox`, but beware that stopping the container sets the sandbox back to its initial state, which removes any changes you made or contracts or tokens that you created.
 
-This tutorial uses an NFT collection contract. Each time the user mints a new set
-(collection) of tokens, a new NFT contract is created. The user cannot add more
-tokens or remove (burn) existing tokens within the contract. However, tokens can
-be transferred to other owners.
+   Unlike the live Tezos networks, this sandbox bakes a new block every 5 seconds.
+   Therefore, commands that you run on the sandbox can take a few seconds to complete.
 
-`mint` command requires the following parameters:
+## Create NFT metadata
 
-- <owner> alias or address of the new token's owner
-- `--tokens` new tokens metadata. Each token metadata is represented as comma
-  delimited string: `'<token_id>, <token_symbol>, <token_name>'`:
+The first step in creating NFTs is to create local metadata files that describe the collection and the individual NFTs:
 
-```sh
-$ tznft mint <owner_alias> --tokens <token_meta_list>`
-```
+1. Create a collection metadata file by running this command:
 
-Example:
+   ```bash
+   tznft create-collection-meta my_collection
+   ```
 
-```sh
-$ tznft mint bob --tokens '0, T1, My Token One' '1, T2, My Token Two'
+   The new metadata file is named `my_collection.json` and has information such as the name, description, home page, and creator of the collection.
+   It also includes the interfaces that the NFTs support, including the TZIP-12 interface that was mentioned earlier.
 
-originating new NFT contract...
-originated NFT collection KT1XP3RE6S9t44fKR9Uo5rAfqHvHXu9Cy7fh
-```
+1. Optional: Edit the `my_collection.json` file to put your information in the `name`, `description`, and `authors` fields.
 
-### Inspecting The NFT Contract
+1. Validate the collection by running this command:
 
-Using `KT1...` address of the NFT contract created by the `mint` command, we can
-inspect token metadata and balances (i. e. which addresses own the tokens).
+   ```bash
+   tznft validate-collection-meta my_collection.json
+   ```
 
-#### Inspect Token Metadata
+   If you did not change values in the file, this command may show warnings that the collection uses placeholder values.
+   You can continue with these placeholder values or insert your own information.
+   If there are any errors, make sure that the file is valid JSON.
 
-`show-meta` command requires the following parameters:
+1. Create a metadata file for the first NFT in the collection by running this command:
 
-- `--nft` address of the FA2 NFT contract to inspect
-- `--signer` alias on behalf of which contract is inspected
-- `--tokens` a list of token IDs to inspect
+   ```bash
+   tznft create-nft-meta Token1 bob ipfs://QmRyTc9KbD7ZSkmEf4e7fk6A44RPciW5pM4iyqRGrhbyvj
```

Comment:
This is a pic of the Tezos logo that was already in there. I don't know if it might be unpinned. Should I update the tutorial to have the user pin some image?